source/colors.js

```diff
@@ -0,0 +1,237 @@
+
```

Comment:
I'm not sure why this file is different than the json block. Or, to put better, I think that that structure is probably all we need, this one seems a bit wordy. Then your generation of the .json file can simply be a `json.stringify()`